src/sql/ggplot/ggplot.py

```diff
@@ -70,9 +70,10 @@ def _draw(self, other) -> mpl.figure.Figure:
                 ax_ = self.figure.add_subplot(n_rows, n_cols, i + 1)
                 facet_key_val = {"key": other.facet, "value": value[0]}
                 self.geom.draw(self, ax_, facet_key_val)
+                handles, labels = ax_.get_legend_handles_labels()
                 ax_.set_title(value[0])
                 ax_.tick_params(axis="both", labelsize=7)
-                ax_.legend(prop={"size": 10})
+                ax_.legend(handles[::-1], labels[::-1], prop={"size": 10})
```

Comment:
What does this change do?

Reply:
It reverses the order of legend so that alphabetically first goes to the top, which matches the order of the stack. This is added to replicate how stack in R works, and I added a comment explaining this to the code.

![histogram_stacked_default](https://github.com/ploomber/jupysql/assets/43221691/51340a71-3729-4ca9-a6e8-055bc5cc26ec)